metriq_gym/exporters/base_exporter.py

```diff
@@ -25,19 +25,31 @@ def as_dict(self):
         record = {
             "app_version": self.metriq_gym_job.app_version,
             "timestamp": self.metriq_gym_job.dispatch_time.isoformat(),
-            "provider": self.metriq_gym_job.provider_name,
             "suite_id": self.metriq_gym_job.suite_id,
-            "device": self.metriq_gym_job.device_name,
             "job_type": self.metriq_gym_job.job_type.value,
-            "results": dict(self.result),
+            "results": {
+                "values": self.result.values,
+                "uncertainties": self.result.uncertainties if self.result.uncertainties else {},
+            },
         }
 
-        platform = {
-            "provider": self.metriq_gym_job.provider_name,
-            "device": self.metriq_gym_job.device_name,
-            "device_metadata": self._derive_device_metadata(),
-        }
-        record["platform"] = platform
+        job_platform = getattr(self.metriq_gym_job, "platform", None)
+        platform_info: dict[str, Any]
+        if isinstance(job_platform, dict):
+            platform_info = {k: v for k, v in job_platform.items() if v is not None}
+        else:
+            platform_info = {}
```

Comment:
[nit]: Minor simplification. 

```suggestion
        platform_info: dict[str, Any] = {}
        if isinstance(job_platform, dict):
            platform_info = {k: v for k, v in job_platform.items() if v is not None}
```